src/modules/repository/implementation/sequelize/migrations/20231221131300-update-abilities.js

```diff
@@ -0,0 +1,91 @@
+const newRoutes = [
+    'V0/PUBLISH',
+    'V0/UPDATE',
+    'V0/GET',
+    'V0/QUERY',
+    'V0/OPERATION_RESULT',
+    'V0/INFO',
+    'V0/BID-SUGGESTION',
+    'V0/LOCAL-STORE',
+];
+const outdatedRoutes = ['PROVISION', 'SEARCH', 'SEARCH_ASSERTION', 'PROOFS'];
+
+async function getAbilityIds(names, queryInterface, transaction) {
+    const [abilities] = await queryInterface.sequelize.query(
+        `SELECT id FROM ability WHERE name IN (${names.map((name) => `'${name}'`).join(', ')})`,
+        { transaction },
+    );
+    return abilities.map((ability) => ability.id);
+}
+
+async function getAdminRoleId(queryInterface, transaction) {
+    const [[role]] = await queryInterface.sequelize.query(
+        "SELECT id FROM role WHERE name='ADMIN'",
```

Comment:
We should also handle the case when there are different roles that are defined by users, not only admin. So we need to maintain their access privileges